dev/cognitiveservices/data-plane/Language/common.json

```diff
@@ -806,7 +831,38 @@
         "sentenceCount": {
           "type": "integer",
           "format": "int32",
-          "description": "It controls the approximate number of sentences in the output summaries."
+          "description": "(Deprecated: Use summaryLength buckets instead) It controls the approximate number of sentences in the output summaries."
+        },
+        "summaryLength": {
+          "type": "string",
+          "enum": [
+            "none",
```

Comment:
What does `none` do as opposed to eliding this property?

Reply:
Good point, removed